datafusion/src/datasource/file_format/parquet.rs

```diff
@@ -367,13 +366,28 @@ mod tests {
     };
 
     use super::*;
+    use crate::datasource::listing::ListingOptions;
     use crate::execution::runtime_env::{RuntimeConfig, RuntimeEnv};
     use arrow::array::{
         BinaryArray, BooleanArray, Float32Array, Float64Array, Int32Array,
         TimestampNanosecondArray,
     };
     use futures::StreamExt;
 
+    #[tokio::test]
+    async fn test_merge_schema() -> Result<()> {
```

Comment:
@alamb Not sure we need a test case here since it gets tested implicitly in the `ParquetExec` tests? If you'd rather have an explicit test case here, then I can use the same utility from those test cases over here.

Reply:
I think coverage at the `ParquetExec` level is more than adequate.